app/replicache/pull/route.ts

```diff
@@ -0,0 +1,15 @@
+import { CustomPullRequest } from '@/lib/replicache.types'
+import { NextResponse, type NextRequest } from 'next/server'
+import { process_pull } from './process-pull'
+
+export async function POST(request: NextRequest) {
+  const body = await request.json()
+  console.log(`Processing pull`, CustomPullRequest.parse(body))
```

Comment:
I know you mention below that error handling is a todo, but `parse` will throw here, which is probably not what we want while trying to log something.

Something like:

```ts
const result = CustomPullRequest.safeParse(body);
console.log("Processing pull", result);

if (!result.success) {
  return NextResponse.json(result.error.issues, { status: 400 });
}

const response = await process_pull(result.data);
```

should be fine for now.

Reply:
Agree! Fix deployed :)